app/controllers/admin/tags_controller.rb

```diff
@@ -2,6 +2,7 @@ module Admin
   class TagsController < Admin::ApplicationController
     layout "admin"
 
+    before_action :badges_for_options
```

Comment:
is this needed for all actions?

Reply:
All of them except `index` needed them because they all rendered the form. I'll make sure to add an `only` clause once I add the `show` action as well (not sure yet if it will need it too but I'm thinking it won't)